### PR TITLE
feat(SwingSet): implement `console` event `source` property

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

As the raw slog data below shows, the `"type":"console"` lines now have a `"source"` of either `"vat"`, `"xsnap"` or `"ls"`.  Also shown is that exceptions are logged to the `"console"` slog, where they weren't before.

```jsonl
{"time":1651791523.6751983,"type":"console","source":"vat","state":"delivery","crankNum":3388,"vatID":"v14","deliveryNum":74,"level":"error","args":["Unregistered update","{ address: 'agoric17xpfvakm2amg962yls6f84z3kell8c5lzyplmt', amount: '200000', denom: 'urun' }"]}
{"time":1651791523.6753771,"type":"console","source":"vat","state":"delivery","crankNum":3388,"vatID":"v14","deliveryNum":74,"level":"error","args":["Unregistered update","{ address: 'agoric1scjvenaehgg4fg5v9ws793fwhqtnsphd62e20f', amount: '13000000', denom: 'ubld' }"]}
{"time":1651791523.675522,"type":"console","source":"vat","state":"delivery","crankNum":3388,"vatID":"v14","deliveryNum":74,"level":"error","args":["Unregistered update","{ address: 'agoric1scjvenaehgg4fg5v9ws793fwhqtnsphd62e20f', amount: '12344800000', denom: 'urun' }"]}
{"time":1651791611.5837805,"type":"console","source":"ls","state":"delivery","crankNum":4087,"vatID":"v6","deliveryNum":4,"level":"info","args":["TypeError#1:","target has no method","foo",", has","[ 'getQuoteIssuer', 'getTimerService', 'makeQuoteNotifier', 'mutableQuoteWhenGT', 'mutableQuoteWhenGTE', 'mutableQuoteWhenLT', 'mutableQuoteWhenLTE', 'quoteAtTime', 'quoteGiven', 'quoteWanted', 'quoteWhenGT', 'quoteWhenGTE', 'quoteWhenLT', 'quoteWhenLTE' ]"]}
{"time":1651791611.5838675,"type":"console","source":"ls","state":"delivery","crankNum":4087,"vatID":"v6","deliveryNum":4,"level":"info","args":["TypeError: target has no method \"foo\", has [\"getQuoteIssuer\",\"getTimerService\",\"makeQuoteNotifier\",\"mutableQuoteWhenGT\",\"mutableQuoteWhenGTE\",\"mutableQuoteWhenLT\",\"mutableQuoteWhenLTE\",\"quoteAtTime\",\"quoteGiven\",\"quoteWanted\",\"quoteWhenGT\",\"quoteWhenGTE\",\"quoteWhenLT\",\"quoteWhenLTE\"]\n at makeError (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/supervisors/subprocess-xsnap/lockdown-subprocess-xsnap.js:2714)\n at fail (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/supervisors/subprocess-xsnap/lockdown-subprocess-xsnap.js:2841)\n at localApplyMethod (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/supervisors/subprocess-xsnap/lockdown-subprocess-xsnap.js:8710)\n at apply ()\n at dispatchToHandler (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/supervisors/subprocess-xsnap/lockdown-subprocess-xsnap.js:8931)\n at (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/supervisors/subprocess-xsnap/lockdown-subprocess-xsnap.js:8564)\n at win (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/supervisors/subprocess-xsnap/lockdown-subprocess-xsnap.js:9274)\n at ()\n"]}
```

### CODE REVIEW

**Comment by warner** [comment]
Could you make it `source: "liveslots"` instead of `ls`? I know it's longer, but it shouldn't see too much output, and I think it'll be easier to use in the long run.

**Review by warner — APPROVED** [review]
With that one change, and fixing the swingset-runner test (@FUDCo any idea what's up there?), this looks good.

File: packages/swingset-runner/test/test-demo.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ async function innerTest(t, extraFlags, dbdir) {
       output += data;
     });
     proc.addListener('exit', code => {
+      t.log(output);
```
Comment (michaelfig) [inline packages/swingset-runner/test/test-demo.js:43]: @FUDCo I made this change to help debug the swingset tests.

Basically, `t.log` doesn't produce any output unless you run `yarn test -v`.  Is that acceptable to you, or should I comment this line out?

Comment (FUDCo) [reply packages/swingset-runner/test/test-demo.js:43]: In this case the test is watching the output from the vat console, but there's no need for that vat console output to appear in the output of the test itself, so the `t.log` may be unnecessary.  But it does no harm.

Comment (michaelfig) [reply packages/swingset-runner/test/test-demo.js:43]: Thanks.

**Pull request merged**